src/main/java/nextstep/security/authorization/RequestMatcherDelegatingAuthorizationManager.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.security.authorization;
+
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+import nextstep.security.matcher.RequestMatcherEntry;
+
+import java.util.List;
+
+public class RequestMatcherDelegatingAuthorizationManager implements AuthorizationManager<HttpServletRequest> {
+    private final List<RequestMatcherEntry<AuthorizationManager>> mappings;
```

Comment:
AuthorizationManager<T>는 제네릭을 통해 해당 AuthorizationManager에서 인가를 위해 활용할 정보를 정의해서 check 시 사용합니다.
AuthorizationManager<RequestAuthorizationContext>에서 RequestAuthorizationContext 코드를 살펴보면 request와 variable이 있습니다.
참고로 말씀드리면 variable은 hasVariable에서 사용하는 값입니다

```java
/**
	* Specify that a path variable in URL to be compared.
	*
	* <p>
	* For example, <pre>
	* requestMatchers("/user/{username}").hasVariable("username").equalTo(Authentication::getName)
	* </pre>
	* @param variable the variable in URL template to compare.
	* @return {@link AuthorizedUrlVariable} for further customization.
	* @since 6.3
	*/
public AuthorizedUrlVariable hasVariable(String variable) {
	return new AuthorizedUrlVariable(variable);
}
```

Reply:
RequestAuthorizationContext는 AuthorizationManager 인가에 사용될 VO 군요!